demos/smartlab_demo/python/object_detection/subdetectors.py

```diff
@@ -0,0 +1,64 @@
+from .preprocess import preprocess
+from .postprocess import postprocess
+from .settings import MwGlobalExp
+from .deploy_util import demo_postprocess
+
+
+class SubDetector:
+    def __init__(self, exp: MwGlobalExp, backend: str='openvino'):
+        self.inode, self.onode, self.input_shape, self.model = exp.get_openvino_model()
+
+        self.cls_names = exp.mw_classes
+        self.num_classes = exp.num_classes
+        self.conf_thresh = exp.conf_thresh
+        self.nms_thresh = exp.nms_thresh
+
+    def inference(self, img):
+        img_info = {"id": 0}
+        height, width = img.shape[:2]
+        img_info["height"] = height
+        img_info["width"] = width
+
+        img_feed, ratio = preprocess(img, self.input_shape)
+        img_info["ratio"] = ratio
+        res = self.model.infer_new_request(inputs={self.inode: img_feed})[self.onode]
```

Comment:
With newer OV it wails with
```
Traceback (most recent call last):
  File "C:\Users\vzlobin\r\z\demos\smartlab_demo\python\smartlab_demo.py", line 154, in <module>
    main()
  File "C:\Users\vzlobin\r\z\demos\smartlab_demo\python\smartlab_demo.py", line 99, in main
    top_det_results, front_det_results = detector.inference(
  File "C:\Users\vzlobin\r\z\demos\smartlab_demo\python\object_detection\detector.py", line 204, in inference
    top_bboxes, top_cls_ids, top_scores = self._detect_one(img_top, view='top')
  File "C:\Users\vzlobin\r\z\demos\smartlab_demo\python\object_detection\detector.py", line 142, in _detect_one
    outputs, img_info = sub_detector.inference(img)
  File "C:\Users\vzlobin\r\z\demos\smartlab_demo\python\object_detection\subdetectors.py", line 24, in inference
    res = self.model.infer_new_request(inputs={self.inode: img_feed})[self.onode]
  File "C:\Users\vzlobin\p\Python\lib\site-packages\openvino\runtime\ie_api.py", line 186, in infer_new_request
    return super().infer_new_request(
RuntimeError: Can't SetBlob with name: images, because model input (shape={1,3,416,416}) and blob (shape=(3.416.416)) are incompatible
```